ocaml/lambda/translclass.ml

```diff
@@ -67,7 +67,9 @@ let lapply ap =
       Lapply ap
 
 let lparam name layout : Lambda.lparam =
-  { name; layout;
+  { name;
```

Comment:
I think at many call-sites it would be possible to provide a good uid with what is already available or by threading through a little more info from the typechecker.

This is also true for many other `internal_not_actually_unique` uses in this file.  I haven't bothered to label them all, because it's not obvious to me how much value there is in improving debugging for the class system.